themes/docdock/static/js/docdock.js

```diff
@@ -204,8 +204,8 @@ jQuery(document).ready(function() {
             location.href = jQuery(this).attr('href');
         });
     });
-
-    jQuery(document).keydown(function(e) {
+    
+    /*jQuery(document).keydown(function(e) {
```

Comment:
Can you add a comment stating why these are commented out? Something like:

```suggestion
    // Commented out to prevent the left/right arrow keys from moving to the left/right pages
    /*jQuery(document).keydown(function(e) {
```